File: templates/python/protection_fault_distribution_shift.py
"""protection_fault_distribution_shift: 保护与故障分析分布漂移（distribution 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='protection and fault analysis: distribution shift'):
    return make_template_figure('distribution', seed=4012, title=title, domain='protection and fault analysis', topic='distribution shift')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)

File: templates/matlab/protection_fault_distribution_shift.m
function fig = protection_fault_distribution_shift()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('distribution', 4012, 'protection and fault analysis: distribution shift', 'protection and fault analysis', 'distribution shift');
end
